ci/packages/components.yml

```diff
@@ -1,4 +1,18 @@
-components build-storybook:
+.run_always_rules: &run_always_rules
+    refs:
+      - develop
+      - releases
+      - schedules
+      - /^release\//
+      - /^run\//
+    changes:
+      - packages/components
+      - packages/components-storybook
+      - yarn.lock
```

Comment:
JFYI we have found this quite problematic at firmware. The `changes` keyword applies only to the latest commit. So if you have two commits and you are not changing these files in the latest but you are in the previous one it will not get triggered.

This is not true if you are using gitlab's Merge Requests but that is not the case for us obviously.

Reply:
I am aware of that, but in this case we do not rely on this "changes" rule. It is just an addition to the manual trigger. Of course we will see if it this setup is gonna suit us.